src/RoutePatches.tsx

```diff
@@ -1,31 +1,61 @@
+import { debounce } from "lodash-es";
 import { afterPatch, RoutePatch, ServerAPI, wrapReactType } from "decky-frontend-lib";
 import { ReactElement } from "react";
-import { AppDetails, AppOverview } from "./app/model";
+import { AppDetails, AppOverview, OverallPlayTimes } from "./app/model";
 import { Mountable } from "./app/system";
 import { Storage } from "./app/Storage";
 import { runInAction } from "mobx";
-//import logger from './utils'
+import logger from './utils'
 
-export function updatePlaytimes(storage: Storage) {
-	storage.getOverallTimes().then(times => {
-		//logger.info("Setting playtimes to app store overviews...");
+//export const updatePlaytimesDelayedDebounced = debounce((storage: Storage, cached: boolean = false) => delay(() => updatePlaytimes(storage, cached), 50, storage), 1000, { leading: true, trailing: false });
+export const updatePlaytimesDebounced = debounce((storage: Storage, cached: boolean = false) => updatePlaytimes(storage, cached), 1000, { leading: true, trailing: false });
```

Comment:
Why do we need debounce here? 
How often does this function triggered? 
Where do the unwanted calls originate from?

Reply:
Debounce is because of "Mount" and "UserLoggedIn" event SteamLifeCycle handlers . They both get executed at startup so simply calling updatePlaytimes() would execute twice. Maybe there is a better way to handle it?